app/models/review.rb

```diff
@@ -12,7 +15,11 @@ class Review < ApplicationRecord
   protected
     def update_average_rating
       @movie = Movie.find(self.movie_id)
-      @movie.rating = @movie.reviews.average(:rating).round(2)
+      if @movie.reviews.present?
```

Comment:
we do we have instance variable in model?

Reply:
Yes sir it works fine.

Reply:
its not about working fine, it should not be there.
use simple local variable here.